packages/frontend/app/styles/components/sidebar.css

```diff
@@ -0,0 +1,13 @@
+a.contact-display{
+    display: inline-block;
+    width: 80%;
```

Comment:
Why always 80%

Would something with dynamic frow be better here? Maybe something using css-grid?

Reply:
Isn't 80% considered dynamic where it will always be 80% of the sidebar? or is it just better/cleaner to use something like a grid?

Reply:
80% just seems arbitrary, wondering why you chose 80%?

but yeah, with grid, things are more concrete and predictable -- and easier to adapt to responsive designs.